Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -53,6 +53,17 @@
 
 #include "UICommon/GameFile.h"
 
+#include "windows.h"
```

Comment:
I am pretty sure *all* of these includes aren't necessary. I'd pare it down to the ones that are necessary.